xml/deployment_cephfs.xml

```diff
@@ -75,6 +75,15 @@
      only be added while the file system is unmounted.
     </para>
    </listitem>
+   <listitem>
+     <para>
+       A minimum of one &mds;. We recommend deploying several
+       nodes within the MDS role. By default additional MDS daemons start
```

Comment:
We typically use the phrase `deploy several nodes with the MDS role`.

Reply:
> We typically use the phrase `deploy several nodes with the MDS role`.

`We recommend deploying several nodes within the MDS role.` 
vs.
`We recommend to deploy several nodes within the MDS role.`

I, personally, recommend the verbiage of the first option. Any particular reason for this?  @tbazant ?